Remotion/ObjectBinding/Web/UI/Controls/Validation/IBocListValidationFailureRepository.cs

```diff
@@ -25,6 +25,17 @@ namespace Remotion.ObjectBinding.Web.UI.Controls.Validation
   /// </summary>
   public interface IBocListValidationFailureRepository
   {
+    /// <summary>
+    /// Delegate which is used for all events that correspond to changes of the validation failures in this <see cref="IBocListValidationFailureRepository"/>.
+    /// </summary>
+    public delegate void ChangedValidationFailures ();
+
+    /// <summary> Raised when <see cref="BusinessObjectValidationFailure"/>s are added to the repository. </summary>
+    /// <remarks>
+    ///   Register for this event to execute code that syncs its validation errors to this <see cref="IBocListValidationFailureRepository"/>.
+    /// </remarks>
+    public event ChangedValidationFailures? ValidationFailureAdded;
```

Comment:
In .NET, events are usually based on EventHandler or EventHandler<T> (for custom argument types). It's best to keep that pattern for symetry/expected API shape.

```suggestion
    public event EventHandler? ValidationFailureAdded;
```
And drop associated custom type.